bin/oio-blob-rebuilder

```diff
@@ -58,6 +58,8 @@ tube for broken chunks events.
                         help="Number of workers (1)")
     parser.add_argument('--chunks-per-second', type=int,
                         help="Max chunks per second per worker (30)")
+    parser.add_argument("--random-wait", type=int,
+                        help="Random wait (in ms)")
```

Comment:
For a better UX, I would use a floating point number of seconds (more pythonic), or an integer number of microseconds (as we use in most of our C programs).